plugin.image.dumpert/addon.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+	id="plugin.image.dumpert"
+	name="Dumpert"
+	version="1.0.4+matrix.1"
+	provider-name="Skipmode A1">
+  <requires>
+    <import addon="xbmc.python"                     version="3.0.0"/>
+	<import addon="script.module.beautifulsoup4"    version="4.3.2"/>
+    <import addon="script.module.requests"          version="2.4.3"/>
+    <import addon="script.module.future"            version="0.0.1"/>
+  	<import addon="script.module.html5lib"          version="0.999.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" 	library="addon.py">
+    <provides>image</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+	<summary lang="en_GB">Watch funny pictures from Dumpert.nl (dutch)</summary>
+	<description lang="en_GB">Watch funny pictures from Dumpert.nl (dutch)</description>
+	<disclaimer lang="en_GB">For bugs, requests or general questions visit the Dumpert.nl thread on the Kodi forum.</disclaimer>
+	<summary lang="nl_NL">Bekijk grappige plaatjes van Dumpert.nl (dutch)</summary>
+	<description lang="nl_NL">Bekijk grappige plaatjes van Dumpert.nl (dutch)</description>
+	<disclaimer lang="nl_NL">Bugs of andere feedback op deze plugin kan geplaatst worden in de Dumpert.nl thread op het Kodi forum.</disclaimer>
+    <language>nl</language>
+    <platform>all</platform>
+    <license>GPL-2.0-or-later</license>
```

Comment:
The license type does not match the on in license.txt  and the plugin of the `krypton` branch?

Reply:
Yeah that's correct, i didn't do pull requests for the licence change. It is changed in my git for my addons however, so next time there is a real change, the licence change will be in the PR.

Reply:
The I would suggest to stick with the original license in the addon.xml and make the complete change in the next PR. Now they are inconsistent, within Matrix, but also between Matrix and other Kodi versions.

Reply:
Do i need to change something?

Reply:
The issue is that the Kodi Matrix version of the Dumpert add-on has a different license than the Dumpert add-on from per-Matrix. They should be the same, as it is the same add-on, just targeted at a new Kodi version. The LICENSE.txt still shows GPL3, just like the pre-Matrix version. So the addon.xml should als have GPL3 targeted: either GPL-3.0-only or GPL-3.0-or-later.

So the addon.xml needs to be updated to reflect the correct license info.

Reply:
addon.xml is changed.